src/docs/tutorials/proximity-steering.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Setting up Proximity Steering using Cloudflare
+---
+
+### What is Proximity Steering?
+
+>Proximity steering routes visitors ... to the closest physical data center.
+
+*Source: [Proximity steering](https://developers.cloudflare.com/load-balancing/understand-basics/traffic-steering/steering-policies/proximity-steering/)*
+
+## About this Tutorial
+
+As Railway does not offer native Proximity Steering at this time, we instead need to place Cloudflare in front of our services to do this for us.
+
+This tutorial aims to provide a simple step-by-step guide on setting everything up on Cloudflare to ensure Proximity Steering works flawlessly!
+
+**Objectives**
+
+In this tutorial, you will learn how to -
+
+- Create a Health monitor.
+- Create pools for each region.
+- Set up the Proximity Load Balancer.
+
+**Prerequisites**
+
+- Have two or more identical services deployed in two or more different regions.
+
+
+    Duplicating a service can be done by right clicking and selecting **Duplicate**, opening its service settings and changing the region, then clicking **Deploy**.
+
+    The services you have deployed into the multiple regions should **only** have a Railway generated domain on them, it also helps to have the domains indicate the region.
+
+    Its recommended to use [shared variables](/guides/variables#shared-variables) or [reference variables](/guides/variables#referencing-another-services-variable) for duplicated services to keep variables in sync.
+
+<Image src="https://res.cloudinary.com/railway/image/upload/v1722015743/docs/tutorials/proximity-load-balancing/region_services_u10ukp.png"
+alt="screenshot of two railway services in different regions"
+layout="responsive"
+width={890} height={435} quality={100} />
+
+- Have your desired domain setup with Cloudflare's nameservers, they have a general guide for that [here](https://developers.cloudflare.com/dns/zone-setups/full-setup/setup/).
+
+- Have **SSL/TLS** mode set to **Full**.
+
+    **SSL/TLS → Overview → Full**
+
+- Have **Always Use HTTPS** enabled.
+    
+    **SSL/TLS → Edge Certificates → Always Use HTTPS**
+    
+    This ensures that Railway avoids managing the insecure redirect, which would otherwise lead to an incorrect redirection to an upstream endpoint.
+
+- Have a `/health` or similar endpoint that returns a status code of 200.
+
+    This allows Cloudflare to check the health of our Railway services so they can handle region failover. As a bonus this can also be used on Railway to achieve [zero-downtime deployments](/reference/healthchecks).
+
+
```

Comment:
Same thought here.. clarify that all of the steps below will be done inside of CloudFlare